blog/forms.py

```diff
@@ -0,0 +1,15 @@
+"""Forms of blog app."""
+
+from django import forms
+
+from .models import Post
+
+
+class PostForm(forms.ModelForm):
+    """Form for Post model."""
```

Comment:
Forms are not for models, but for some logic.

Reply:
Fixed.